duckduckgo/Android
# Allow onboarding to be relaunched

Asana Issue URL: https://app.asana.com/0/488551667048375/528763843660268

## Description
Adds a launch activity to the app (this is needed for tabs as well) and allows onboarding to be relaunched from settings. Also updates the Settings screen to have subheadings.

## Steps to Test this PR:
TEST 1
1. Freshly install the app and ensure onboarding is shown on first launch
1. Finish onboarding
1. Relaunch and ensure it is not displayed again

TEST 2
1. Open Settings
1. Tap "Our features"
1. Check that onboarding is launched again

TEST 3
1. Open Settings
1. Note the headings look ok
